domains/julio.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "My work portfolio",
+    "repo": "https://github.com/thatsquite",
+    "owner": {
+        "username": "thatsquite",
+        "email": "juliocesar.santos@making.com"
+    },
+    "record": {
+        "CNAME": "thatsquite.github.io"
```

Comment:
404 site. Please display some content on a finished site.

Reply:
@DEV-DIBSTER There is a website preview and link in the PR description